### ISSUE TITLE
BF4 scenario fails with No route available from PFS

### ISSUE THREAD

**Issue by agatsoh** [raiden-network/raiden#5337]
## Problem Definition
Scenario in a nutshell
nodes:
node 0: `0x0c02eB803103BC69Bbe35f773E1C1aeDd41B3240`
node 1: `0x979cBC13628F59B8158bFd74F4C1C5BC135d6b76`
node 2: `0xCE8Ec7F8ccFf40f2CB7A8AfAc3f34f8675381c3A`
node 3: `0x1B051dc7f9709775F3C19523bDFE9f46dF6A6dBF`
node 4: `0xfA034670eDF9AD1525468e43E61d8a138A26f6FA`
node 5: `0xdf9eb27A724812A61D91f55dD11f4a3f39b481DC`

Scenario log error

```
{
  "task": "<ParallelTask: {'name': 'Make 100 transfers from 0 to 2, 3 and 5 in parallel', 'tasks': [{'serial': {'name': 'Make 100 transfers from 0 to 2', 'repeat': 100, 'tasks': [{'transfer': {'from': 0, 'to': 2, 'amount': 1000000000000000, 'lock_timeout': 30}}]}}, {'serial': {'name': 'Make 100 transfers from 0 to 3', 'repeat': 100, 'tasks': [{'transfer': {'from': 4, 'to': 3, 'amount': 1000000000000000, 'lock_timeout': 30}}]}}, {'serial': {'name': 'Make 100 transfers from 0 to 5', 'repeat': 100, 'tasks': [{'transfer': {'from': 4, 'to': 5, 'amount': 1000000000000000, 'lock_timeout': 30}}]}}]}>",
  "event": "Task errored",
  "logger": "scenario_player.tasks.base",
  "level": "error",
  "timestamp": "2019-11-21 07:56:30.534461",
  "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 74, in __call__\n    return self._run(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/execution.py\", line 65, in _run\n    group.join(raise_error=True)\n  File \"/usr/local/lib/python3.7/site-packages/gevent/pool.py\", line 433, in join\n    greenlet._raise_exception()\n  File \"src/gevent/greenlet.py\", line 312, in gevent._greenlet.Greenlet._raise_exception\n  File \"/usr/local/lib/python3.7/site-packages/gevent/_compat.py\", line 47, in reraise\n    raise value.with_traceback(tb)\n  File \"src/gevent/greenlet.py\", line 716, in gevent._greenlet.Greenlet.run\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 74, in __call__\n    return self._run(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/execution.py\", line 41, in _run\n    task()\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/base.py\", line 74, in __call__\n    return self._run(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/scenario_player/tasks/api_base.py\", line 59, in _run\n    f'HTTP status code \"{resp.status_code}\" while fetching {url}. '\nscenario_player.exceptions.legacy.RESTAPIStatusMismatchError: HTTP status code \"409\" while fetching http://127.0.0.1:59829/api/v1/payments/0x62083c80353Df771426D209eF578619EE68D5C7A/0x1B051dc7f9709775F3C19523bDFE9f46dF6A6dBF. Expected 2..: {\"errors\": \"Payment couldn't be completed because: there is no route available\"}"
}
```
what does the exception say:

1. Stage where it fails:
```
[
  {
    "serial": {
      "name": "Make 100 transfers from 0 to 2",
      "repeat": 100,
      "tasks": [
        {
          "transfer": {
            "from": 0,
            "to": 2,
            "amount": 1000000000000000,
            "lock_timeout": 30
          }
        }
      ]
    }
  },
  {
    "serial": {
      "name": "Make 100 transfers from 0 to 3",
      "repeat": 100,
      "tasks": [
        {
          "transfer": {
            "from": 4,
            "to": 3,
            "amount": 1000000000000000,
            "lock_timeout": 30
          }
        }
      ]
    }
  },
  {
    "serial": {
      "name": "Make 100 transfers from 0 to 5",
      "repeat": 100,
      "tasks": [
        {
          "transfer": {
            "from": 4,
            "to": 5,
            "amount": 1000000000000000,
            "lock_timeout": 30
          }
        }
      ]
    }
  }
]
```

2. Request which fails:
`http://127.0.0.1:59829/api/v1/payments/0x62083c80353Df771426D209eF578619EE68D5C7A/0x1B051dc7f9709775F3C19523bDFE9f46dF6A6dBF`

where:
`0x62083c80353Df771426D209eF578619EE68D5C7A`: token address
`0x1B051dc7f9709775F3C19523bDFE9f46dF6A6dBF`: node 3 address

3. Topology of bf4 `[ [0, 1, 2], [0, 1, 3], [0, 4, 5] ]` 
So According to the above request transfers initiated from 
node 4 to 3 fail

4. node4 logs 
```
{
  "errors": "Payment couldn't be completed because: there is no route available",
  "status_code": 409,
  "event": "Error processing request",
  "logger": "raiden.api.rest",
  "level": "error",
  "timestamp": "2019-11-21 07:27:46.341442"
}
```

[bf4_multi_payments_same_node_2019_11_21.tar.gz](https://github.com/raiden-network/raiden/files/3874347/bf4_multi_payments_same_node_2019_11_21.tar.gz)

[SP 5]

**Comment by karlb** [comment]
I assume this is fixed by the wait in https://github.com/raiden-network/raiden/pull/5339. If we don't see this again today, we should close this ticket.

**Comment by Dominik1999** [comment]
Closed and merged into META Issue #5433
